win32ss/user/user32/lang/pt-BR.rc

```diff
@@ -60,9 +60,9 @@ END
 STRINGTABLE
 BEGIN
     IDS_ERROR "Erro"
-    IDS_NOT_RESPONDING " (Not Responding)"
-    IDS_ASK_TERMINATE "This application is hung up. May I terminate this app?"
-    IDS_HUNG_UP_TITLE "Hung up!"
+    IDS_NOT_RESPONDING " (Não respondendo)"
+    IDS_ASK_TERMINATE "Este programa não está respondendo. Você quer finalizar esse programa?"
```

Comment:
"esse" or "este" ?

Reply:
Este, pois ele estaria mais próximo do ser "falante" (no caso do próprio sistema) e pelo fato de estar no falando que o programa não está respondendo no tempo atual, e não no passado ou futuro.

Reply:
I was talking about the second one "Você quer finalizar esse programa?"

Reply:
In this case can be used the "esse".